sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -0,0 +1,306 @@
+// ----------------------------------------------------------------------------
```

Comment:
This isn't really true anymore - can be removed, as the provider will most likely need to be maintained by hand until v2 of unified monitors.